content/docs/tools/pack/concepts/trusted_builders.md

```diff
@@ -9,22 +9,18 @@ From version [0.11.0](https://github.com/buildpacks/pack/releases/tag/v0.11.0) o
 Read the [announcement](https://medium.com/buildpacks/faster-more-secure-builds-with-pack-0-11-0-4d0c633ca619).
 
 ## What is a Trusted Builder?
-
-When `pack` considers a builder to be trusted, `pack build` operations will use a single lifecycle binary called the `creator`. 
+When `pack` considers a builder to be trusted, `pack build` operations will use a single lifecycle binary called the `creator`.
 
 If `pack` doesn't trust a builder it will continue to execute five separate lifecycle binaries: `detect`, `analyze`, `restore`, `build` and `export`. It will run the more privileged phases using a published [lifecycle image][lifecycle-image], if one is available.
 
-This choice strikes a balance between security and performance. The `analyze`, `restore` and `export` phases of the lifecycle require higher levels of privilege or access to sensitive data that is not required by the `detect` and `build` phases. 
+This choice strikes a balance between security and performance. The `analyze`, `restore` and `export` phases of the lifecycle require higher levels of privilege or access to sensitive data that is not required by the `detect` and `build` phases.
 
 ## Why use a Trusted Builder?
```

Comment:
Looks like instead of adding back lines, you're taking them out... 😝 
https://github.com/buildpacks/pack/pull/828#discussion_r481366089